### PR TITLE
Remove yuanmwang-wf from dependabot config

### PR DESCRIPTION
**Pull request by yuanmwang-wf**
^

@Workiva/service-platform

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
<details>
<summary>(1) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>aiohttp < 3.7.4</code> via <code>lib/python/requirements_dev_asyncio.txt</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=frugal">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by matthewschlue-wf** [comment]
You may want to update the default reviewer in the PR template too https://github.com/Workiva/frugal/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -9,7 +9,6 @@ updates:
       - "charliekump-wf"
       - "erikpetersen-wf"
```
Comment (natewoods-wf) [inline .github/dependabot.yaml:10]: 👋 Hey Yuan, Erik is no longer on the team, would you mind removing him from here as well?

**Review by natewoods-wf — APPROVED** [review]

**Comment by natewoods-wf** [comment]
@Workiva/release-management-p for merge

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**